packages/react/src/internal/field-label-wrapper/field-label-wrapper.tsx

```diff
@@ -3,17 +3,17 @@ import React from 'react';
 
 /**
  * Wrapper for `Input` and `Textarea` components, using `FieldLabel` when
- * children is provided.
+ * `label` prop is provided.
  */
 export const FieldLabelWrapper = ({
   id,
-  children: { element, children },
+  children: { element, label },
 }: FieldLabelWrapperProps): JSX.Element => {
-  if (!children) return element;
+  if (!label) return element;
 
   return (
     <FieldLabel htmlFor={id}>
-      <span>{children}</span>
+      <span>{label}</span>
```

Comment:
Remind me again why we need the `<span>`? 🤔

Reply:
It's because of the [flow in component](https://github.com/onfido/castor/blob/feat/input-textarea-label/packages/core/src/components/field-label/field-label.scss#L16), we expect usually a single label (e.g. just a string), and optional element.

For such cases where label is of multiple elements, we need to wrap in `<span>` to make it one element.